### PR TITLE
BIG-199 Add unsupported in filters

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: BIG-199
Tests: https://github.com/keboola/storage-api-php-client/pull/1242
Driver: https://github.com/keboola/php-storage-driver-bigquery/pull/103
KBC: https://github.com/keboola/connection/pull/4775

### CODE REVIEW

File: src/Handler/Table/Preview/PreviewTableHandler.php [hunk 1]
```diff
@@ -169,4 +179,46 @@ private function validateFilters(PreviewTableCommand $command): void
...
+    public function getTypesUnsupportedInFilters(): array
+    {
+        return [
+            Bigquery::TYPE_ARRAY,
+            Bigquery::TYPE_STRUCT,
+            Bigquery::TYPE_BYTES,
+            Bigquery::TYPE_GEOGRAPHY,
+            Bigquery::TYPE_INTERVAL,
+            Bigquery::TYPE_JSON,
+        ];
```
Comment (zajca) [inline src/Handler/Table/Preview/PreviewTableHandler.php]: To by mohla byt konstanta klidně

**Review by zajca — CHANGES_REQUESTED** [review]
Chtělo by to přidat test.

File: src/Handler/Table/Preview/PreviewTableHandler.php (hunk 1 above)
Comment (tomasfejfar) [reply src/Handler/Table/Preview/PreviewTableHandler.php]: Done.

File: tests/functional/UseCase/Table/PreviewTableTest.php [hunk 2]
```diff
@@ -1532,6 +1558,36 @@ public function filterProvider(): Generator
             ],
             'Invalid filter value, expected:"INT64", actual:"STRING".',
         ];
+
+        foreach ([
```
Comment (tomasfejfar) [inline tests/functional/UseCase/Table/PreviewTableTest.php:1562]: test tu

**Review by zajca — APPROVED** [review]

**Pull request merged**